### DISCUSSION TITLE
Misalignment between breadcrumbs in Simple and Advanced view

### DISCUSSION THREAD

**Discussion by uslss** [dbeaver/dbeaver#8890] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version win10 x64
- DBeaver version 7.1.0 

#### Describe the problem you're observing:
Breadcrumbs are shown differently in Simple and Advanced view
![image](https://user-images.githubusercontent.com/31996417/83746248-7ef43f80-a667-11ea-959c-1aee9a511ede.png)

**Comment by serge-rider** [comment, marked as answer]
This is not a bug. Simple view has different database metadata structure (simplified). 
So it works as expected.
